src/toga_chart/chart.py

```diff
@@ -44,6 +44,12 @@ def __init__(
 
         super().__init__(id=id, style=style)
 
+        if not self._impl:
+            # Backwards compatibility for Toga 0.4.8, which does not call the _create()
+            # method in Widget.__init__()
+            self._create()
```

Comment:
We can also fix this by a toga dependency to `toga-core>=0.4.8`. It's an interesting omission that we don't currently have that dependency, only enforcing the `toga-dummy` requirement on developers.

Reply:
Oh. Yeah. That is a good point! I'll just change it to that once we know for sure what the next version number is.

Reply:
I've removed the backwards compatibility shim and added the version requirement (which fails for now, of course, since Toga 0.5.0 doesn't exist).